Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -47,54 +47,57 @@ def __init__(self, app_id: str, subscription_id: str, resource_group_name: str,
         self.connection_type = connection_type
 
     @logger
-    def storage_account_list_request(self, account_name: str) -> Dict:
+    def storage_account_list_request(self, account_name: str, resource_group_name: str, subscription_id: str) -> Dict:
         """
             Send the get storage account/s request to the API.
         Args:
             account_name: The storage account name, optional.
+            resource_group_name: The resource group name.
+            subscription_id: The subscription id.
 
         Returns:
             The json response from the API call.
         """
 
         return self.ms_client.http_request(
             method='GET',
-            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
-                       f'{account_name}',
-            params={
-                'api-version': API_VERSION,
-            }
+            full_url=f'https://management.azure.com/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.Storage/\
+storageAccounts/{account_name}?api-version={API_VERSION}'
         )
 
     @logger
-    def storage_blob_service_properties_get_request(self, account_name: str) -> Dict:
+    def storage_blob_service_properties_get_request(self, account_name: str,
+                                                    resource_group_name: str,
+                                                    subscription_id: str) -> Dict:
         """
             Send the get blob service properties request to the API.
         Args:
             account_name: The storage account name.
+            resource_group_name: The resource group name.
+            subscription_id: The subscription id.
 
         Returns:
             The json response from the API call.
         """
         return self.ms_client.http_request(
             method='GET',
-            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
-                       f'{account_name}/blobServices/default',
-            params={
-                'api-version': API_VERSION,
-            }
+            full_url=f'https://management.azure.com/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/\
+providers/Microsoft.Storage/storageAccounts/{account_name}/blobServices/default?api-version={API_VERSION}'
         )
 
     @logger
-    def storage_account_create_update_request(self, args: Dict) -> Dict:
+    def storage_account_create_update_request(self, subscription_id: str, resource_group_name: str, args: Dict) -> Dict:
```

Comment:
it is a bit strange that arg is transferred here. Why not transfer the account_name as happened in all the other request functions??